### PR TITLE
nav.about.teams: Create Team page and add to the About nav dropdow

### PR DESCRIPTION
**Pull request by cwhite911**
## About Us > Team (Issue:  #414 )
I added **Team** to the **About Us** nav dropdown and started working on the team page.  I also changed the nav title from **About** to **About Us** and moved it to the first item on the navbar.

TODO:

- [x] Add maintainers (Move to another PR)
- [x] Fix CSS issue with team layout
- [x] Get approved images from everyone.
- [x] Edit descriptive text

## PSC Description:
Screenshot 2024-03-13 at 18-22-00 GRASS GIS - Bringing advanced geospatial technologies to the world

## PSC Chair:
Screenshot 2024-03-13 at 18-22-19 GRASS GIS - Bringing advanced geospatial technologies to the world

## PSC Members:
Screenshot 2024-03-13 at 18-22-43 GRASS GIS - Bringing advanced geospatial technologies to the world

### CODE REVIEW

**Comment by echoix** [comment]
Do you know if the site builder rerasterises/crops the pictures to not have to load a multimegabyte image that will not be fully displayed and bigger than necessary?

**Comment by neteler** [comment]
> Do you know if the site builder rerasterises/crops the pictures to not have to load a multimegabyte image that will not be fully displayed and bigger than necessary?

Generally, there seems to be some magic available in hugo: https://gohugo.io/content-management/image-processing/

**Comment by cwhite911** [comment]
I've finished the setting up the images so that they are resized and filled dynamically. I also added in social info and rearranged the layout.

Screenshot 2024-03-14 at 12-14-41 GRASS GIS - Bringing advanced geospatial technologies to the world
Screenshot 2024-03-14 at 12-14-10 GRASS GIS - Bringing advanced geospatial technologies to the world

**Comment by cwhite911** [comment]
Updated layout

Screenshot 2024-03-14 at 12-24-03 GRASS GIS - Bringing advanced geospatial technologies to the world

**Comment by echoix** [comment]
Would you all be wanting to have the website-in-progress be targeted at a different branch, and that branch be deployed with github-pages? Or have it written so it could work on one of your forks..

**Comment by cwhite911** [comment]
This layout is ready for review.

PSC members please complete the following tasks:

1. Add your photo to `theme/grass/assets/images/team/<FirstName_LastName_PSC>.(png|jpg|webp)`
2. Update your record in `data/team.yml` with the proper image name. The only thing that may need to change is the file type.

```yml
- name: Markus Neteler
    chair: former
    terms: 2006 - 2012, 2012 - 2016, 2016 - 2020
    psc: true
    image: /images/team/Markus_Neteler_PSC.png  # Check here
    website: https://www.mundialis.de/
    job: mundialis, Bonn, DE
    country: Germany
    social:  # Add, Remove, and Update Social                                                                   
      - icon: fa fa-envelope
        link: mailto:neteler@osgeo.org # Check here
      - icon: fab fa-github
        link: https://github.com/neteler # Check here
      - icon: fab fa-linkedin
        link: https://www.linkedin.com/in/markusneteler/  # Check here
      - icon: fab fa-x-twitter
        link: https://twitter.com/MarkusNeteler # Check here
      - icon: fab fa-mastodon
        link: https://mastodon.social/@neteler # Check here
```

3.  Make sure the social accounts you want link are linked. I pulled some info, but please double check it is correct.

@veroandreo please look at the chair section. I added the bio for you from the SMathHacks event.

The learn more about the PSC section could also be part of the Governance page. That is also where we can include other information from https://trac.osgeo.org/grass/wiki/PSC.

**Comment by cwhite911** [comment]
> Would you all be wanting to have the website-in-progress be targeted at a different branch, and that branch be deployed with github-pages? Or have it written so it could work on one of your forks..

I'm not sure. Either works from my perspective, but I am happy to do whatever has worked for the project in the past.

**Review by veroandreo — COMMENTED** [review]
I like the layout. I just cut my description a bit (maybe not needed at all), I feel a bit like an impostor being highlighted there :facepalm: 

I'd move the images into static/images/team, because I don't feel we as team are part of the theme :)

**Comment by cwhite911** [comment]
@echoix let's go with publishing a website-in-progress branch to gh-pages.

> I like the layout. I just cut my description a bit (maybe not needed at all), I feel a bit like an impostor being highlighted there 🤦

lol @veroandreo you are really going to like that I add Dr. Verónica Andreo. It's the right place to :muscle:


> I'd move the images into static/images/team, because I don't feel we as team are part of the theme :)

I followed the docs for image processing when I placed the images in `themes/grass/assests/images/team/`  I tried moving them to `static/images/team`, but `{{ $original := resources.Get .image }}` stops working.

I'll look at it again later, but I think if we want to move the images out of assets we will need to use something like  `{{ $image := resources.GetRemote .Site.BaseURL"/images/team/<image>" }}` or `{{ $image := .Resources.GetMatch <image> }}`. Let me know if you already know how to resolve this with Hugo.

`

**Comment by echoix** [comment]
So I've got it working (on my branch): https://echoix.github.io/grass-website/

I made it so that we'd need to target all the PRs for the new website on branch "preview" instead of "master", so the existing website won't be broken. So while developing the new website, you could merge to the "preview" branch without problems.

**Comment by petrasovaa** [comment]
Looking at the current state, the former members should be at the very bottom. Also if the chair is going to be singled out at the very top, I would suggest then just 2 rows with 4 each. I would probably make all the images smaller. I did some testing on my side how it would look like, and I think it will work, but I don't understand the css well enough at this point.

**Comment by cwhite911** [comment]
So something more like this.

Screenshot 2024-03-15 at 09-52-25 GRASS GIS - Bringing advanced geospatial technologies to the world

**Comment by petrasovaa** [comment]
> So something more like this.

Yes, and perhaps Vero could be a bit smaller as well:)

**Comment by neteler** [comment]
Looking at this part I have a suggestion:

image

As such it may look like "only 1 former member in the PSC" (for those who don't read carefully) - what about adding all PSC members of the respective periods? The images are there anyway and then it looks complete.

Of course the title would not be "Former members" but "PSC 20xx-20yy".

**Comment by veroandreo** [comment]
> > So something more like this.
> 
> Yes, and perhaps Vero could be a bit smaller as well:)

yes, please! PSC is actually an horizontal system, so too much chair highlighting feels a bit uncomfortable to me...

**Comment by cwhite911** [comment]
> As such it may look like "only 1 former member in the PSC" (for those who don't read carefully) - what about adding all PSC members of the respective periods? The images are there anyway and then it looks complete.

We can do this I just need to know who was part of the the PSC for each period. Here is an example using @neteler record.

```yaml
 - name: Dr. Markus Neteler
    psc:
        status: active
        role: member
        periods:
          - start: 2021
            end: 2025
          - start: 2016
            end: 2020
            role: chair
          - start: 2012
            end: 2016
            role: chair
          - start: 2006
            end: 2012
            role: chair
        chair:
          terms: 2006 - 2012, 2012 - 2016, 2016 - 2020
    image: /images/team/Markus_Neteler_PSC.jpg
    website: https://www.mundialis.de/
    job: mundialis, Bonn, DE
    country: Germany
    social:
      - icon: fa fa-envelope
        link: mailto:neteler@osgeo.org
      - icon: fab fa-github
        link: https://github.com/neteler
      - icon: fab fa-linkedin
        link: https://www.linkedin.com/in/markusneteler/
      - icon: fab fa-x-twitter
        link: https://twitter.com/MarkusNeteler
      - icon: fab fa-mastodon
        link: https://mastodon.social/@neteler

```

**Comment by cwhite911** [comment]
> yes, please! PSC is actually an horizontal system, so too much chair highlighting feels a bit uncomfortable to me...

We can make it smaller, but I do think it is nice featuring whoever the active chair is. The whole point of the page is to make it easier to for users to connect and learn about the people behind GRASS.

**Comment by wenzeslaus** [comment]
Tracking `periods` here seems like little too much, but perhaps okay. However, having the former members with all the details definitely seems like too much. Names only? With images, but on a separate page?

**Comment by cwhite911** [comment]
My other thought was adding a dropdown selector to pick the period you are interested in but that is probably overkill.

Here is the page as it stands now.

Screenshot 2024-03-15 at 14-40-00 GRASS GIS - Bringing advanced geospatial technologies to the world

**Comment by neteler** [comment]
> We can do this I just need to know who was part of the the PSC for each period.

I have completed the records here: https://trac.osgeo.org/grass/wiki/PSC#FormerPSCmembers

**Comment by petrasovaa** [comment]
> Tracking `periods` here seems like little too much, but perhaps okay. However, having the former members with all the details definitely seems like too much. Names only? With images, but on a separate page?

I would merge all former members under "Former members" and exclude the current, right now Markus is there as former member which is confusing. I would link the wiki page with all the details.

File: themes/grass/assets/css/style.css [hunk 1]
```diff
@@ -1390,3 +1390,18 @@ code{
...
+}
+
+.member img {
+  border-radius: 50%;
+  object-fit: cover;
+}
+
+.psc-chair {
+  border: 4px solid var(--grass-color-dark);
+}
```
[the remark this answers is not in the document]
Comment (cwhite911) [reply themes/grass/assets/css/style.css:1407]: > Should this distinguish chair and regular member? Markus as former chair has this too and it looks confusing

That was the intention, but we could make it just for the the active chair or even remove it completely.

[the remark this answers is not in the document]
Comment (veroandreo) [reply themes/grass/assets/css/style.css:1407]: I'd remove it

**Comment by cwhite911** [comment]
Screenshot 2024-03-19 at 14-18-26 GRASS GIS - Bringing advanced geospatial technologies to the world

File: themes/grass/layouts/about/team.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                    <h4>Meeting Agenda</h4>
+                    
+                    <p>The PSC meticulously maintains an <a href="https://trac.osgeo.org/grass/wiki/PSC/Agenda" target="_blank">Agenda</a> that outlines dates for important issues requiring votes or further discussion. This ensures transparency and structured decision-making within our project.</p>
+                    
+                    <h4>RFCs and Motions</h4>
+                    
+                        <h5>Request For Comment (RFC) List</h5> 
+                        <p>Dive into the <a href="https://trac.osgeo.org/grass/wiki/RFC" target="_blank">RFCs</a> to explore proposed changes or enhancements to the project. The list is also accessible on <a href="https://github.com/OSGeo/grass/tree/main/doc/development/rfc" target="_blank">GitHub</a>, providing a platform for community feedback and collaboration.</p>
+                        <h5>PSC Motions</h5>
+                        <p>Review the formal decisions and <a href="https://trac.osgeo.org/grass/wiki/PSC/Motions" target="_blank">motions</a> passed by the PSC, reflecting our project's evolving needs and objectives.</p>
```
Comment (petrasovaa) [inline themes/grass/layouts/about/team.html:95]: https://grasswiki.osgeo.org/wiki/PSC#Motions_since_2020

File: themes/grass/layouts/about/team.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+            <div class="p-5 bg-white">
+                <h2>Learn more about the PSC</h2>
+                <p>The Project Steering Committee (PSC) plays a pivotal role in the governance and operational direction of GRASS GIS. Here's a brief overview of the key responsibilities and resources managed by the PSC:</p>
+                    
+                    <h4>Mailing List</h4>
+                    <p>Stay connected and informed with the PSC's activities through our <a href="http://lists.osgeo.org/mailman/listinfo/grass-psc" target="_blank">Public Mailing List</a>. It's the best way to keep up with the discussions, decisions, and directions taken by the committee.</p>
+                    
+                    <h4>Meeting Agenda</h4>
+                    
+                    <p>The PSC meticulously maintains an <a href="https://trac.osgeo.org/grass/wiki/PSC/Agenda" target="_blank">Agenda</a> that outlines dates for important issues requiring votes or further discussion. This ensures transparency and structured decision-making within our project.</p>
```
Comment (petrasovaa) [inline themes/grass/layouts/about/team.html:88]:
```suggestion
                    <p>The PSC holds regular meetings and keeps <a href="https://grasswiki.osgeo.org/wiki/PSC#PSC_meeting_agenda_and_minutes" target="_blank">meeting minutes</a> that summarize discussed issues and decisions. This ensures transparency and structured decision-making within our project.</p>
```

File: themes/grass/layouts/about/team.html [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+                        <h5>Request For Comment (RFC) List</h5> 
+                        <p>Dive into the <a href="https://trac.osgeo.org/grass/wiki/RFC" target="_blank">RFCs</a> to explore proposed changes or enhancements to the project. The list is also accessible on <a href="https://github.com/OSGeo/grass/tree/main/doc/development/rfc" target="_blank">GitHub</a>, providing a platform for community feedback and collaboration.</p>
+                        <h5>PSC Motions</h5>
+                        <p>Review the formal decisions and <a href="https://trac.osgeo.org/grass/wiki/PSC/Motions" target="_blank">motions</a> passed by the PSC, reflecting our project's evolving needs and objectives.</p>
+                    
+                    <h4>Developer Contributions</h4>
+                    
+                    <p>A core function of the PSC is to <a href="https://trac.osgeo.org/grass/wiki/PSC/RequestingSVNWriteAccess" target="_blank">grant GitHub write access</a> to new core or addon developers. This process ensures that contributors who have demonstrated commitment and quality in their work are able to more directly influence the project's development.</p>
```
Comment (petrasovaa) [inline themes/grass/layouts/about/team.html:99]:
```suggestion
                    <p>A core function of the PSC is to grant GitHub write access to new core developers. This process ensures that contributors who have demonstrated commitment and quality in their work are able to more directly influence the project's development.</p>
```

File: themes/grass/layouts/about/team.html (hunk 2 above)
Comment (petrasovaa) [inline themes/grass/layouts/about/team.html:95]:
```suggestion
                        <p>Review the formal decisions and <a href="https://grasswiki.osgeo.org/wiki/PSC#Motions" target="_blank">motions</a> passed by the PSC, reflecting our project's evolving needs and objectives.</p>
```

File: themes/grass/layouts/about/team.html [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+                    
+                    <h4>Meeting Agenda</h4>
+                    
+                    <p>The PSC holds regular meetings and keeps <a href="https://grasswiki.osgeo.org/wiki/PSC#PSC_meeting_agenda_and_minutes" target="_blank">meeting minutes</a> that summarize discussed issues and decisions. This ensures transparency and structured decision-making within our project.</p>
+                    
+                    <h4>RFCs and Motions</h4>
+                    
+                        <h5>Request For Comment (RFC) List</h5> 
+                        <p>Dive into the <a href="https://trac.osgeo.org/grass/wiki/RFC" target="_blank">RFCs</a> to explore proposed changes or enhancements to the project. The list is also accessible on <a href="https://github.com/OSGeo/grass/tree/main/doc/development/rfc" target="_blank">GitHub</a>, providing a platform for community feedback and collaboration.</p>
```
Comment (veroandreo) [inline themes/grass/layouts/about/team.html:93]:
```suggestion
                        <p>The GRASS team uses RFC documents as tools to collectively delineate different procedures and  functions. The list of RFC documents that were adopted is accessible on <a href="https://github.com/OSGeo/grass/tree/main/doc/development/rfc" target="_blank">GitHub</a>, providing a platform for community feedback and collaboration.</p>
```

File: data/team.yml [hunk 6]
```diff
@@ -0,0 +1,420 @@
...
+        role: member
+        periods:
+          - start: 2012
+            end: 2016
+          - start: 2016
+            end: 2020
+          - start: 2020
+            end: 2024
+    image: /images/team/Martin_Landa_PSC.jpg
+    website: https://geo.fsv.cvut.cz/~landa/
```
Comment (petrasovaa) [inline data/team.yml:118]:
```suggestion
    website: https://geomatics.fsv.cvut.cz/en/employees/martin-landa/
```

File: data/team.yml [hunk 7]
```diff
@@ -0,0 +1,420 @@
...
+  - name: Dr. Helmut Kudrnovsky
+    psc:
+        status: active
+        role: member
+        periods:
+          - start: 2020
+            end: 2024
+    image: /images/team/Helmut_Kudrnovsky_PSC.png
+    email: hellik@web.de
+    website: https://www.umweltbundesamt.at/
```
Comment (petrasovaa) [inline data/team.yml:205]:
```suggestion
    website: https://www.osgeo.org/member/kudrnovsky/
```

File: data/team.yml [hunk 8]
```diff
@@ -0,0 +1,420 @@
...
+          - start: 2016
+            end: 2020
+            role: chair
+          - start: 2020
+            end: 2024
+            role: member
+        chair:
+          terms: 2006 - 2012, 2012 - 2016, 2016 - 2020
+    image: /images/team/Markus_Neteler_PSC.jpg
+    website: https://www.mundialis.de/
```
Comment (petrasovaa) [inline data/team.yml:49]:
```suggestion
    website: https://www.mundialis.de/team/dr-markus-neteler/
```

**Review by petrasovaa — APPROVED** [review]

**Pull request merged**